en/mkdocs.yml

```diff
@@ -1583,6 +1583,10 @@ nav:
                 - Manage Applications:
                     - Manage Applications: install-and-setup/setup/api-controller/managing-applications/managing-applications.md
                     - Migrate Apps to Different Environments: install-and-setup/setup/api-controller/managing-applications/migrating-applications-to-different-environments.md
+                - Manage Common API Policies:
+                    - Manage Common API Policies: install-and-setup/setup/api-controller/managing-common-api-policies/managing-common-api-policies.md
+                    - Import Common API Policy Via Dev First Approach: install-and-setup/setup/api-controller/managing-common-api-policies/importing-common-api-policies-via-dev-first-approach.md
```

Comment:
No need of this doc.

Reply:
Fixed by [6465f5a](https://github.com/wso2/docs-apim/pull/6103/commits/6465f5a2f5f93d8ca2645700a95ab612cd0d25ed)